fastapi_sqla/sqla.py

```diff
@@ -146,11 +165,14 @@ def get_users(session: fastapi_sqla.Session):
     return response
 
 
-class SessionDependency:
+S = TypeVar("S", bound=SqlaSession)
+
+
+class SessionDependency(Generic[S]):
```

Comment:
Sorry, I'm very late to the party. I'm curious how come this was needed since `SqlaSession` is overloaded depending on when SQLModel is imported or not 🤔 

Very nice feature btw! 🙌

Reply:
I changed how it was done and forgot to remove this. good catch, will cleanup

Reply:
(Before, there were 2 session types `SqlaSession` & `SQLModelSession`)

Reply:
Got it, thanks for the explanation!

Reply:
fixed in #121